src/src/rx_main.cpp

```diff
@@ -538,16 +543,14 @@ void ICACHE_RAM_ATTR HWtimerCallbackTock()
     bool didFHSS = HandleFHSS();
     bool tlmSent = HandleSendTelemetryResponse();
 
-    #if defined(RADIO_SX127X)
-    if (!didFHSS && !tlmSent && LQCalc.currentIsSet())
+    if (!didFHSS && !tlmSent && LQCalc.currentIsSet() && Radio.FrequencyErrorAvailable())
     {
         HandleFreqCorr(Radio.GetFrequencyErrorbool());      // Adjusts FreqCorrection for RX freq offset
+    #if !defined(Regulatory_Domain_ISM_2400)
```

Comment:
this should be `#if defined(RADIO_SX127X)`

Reply:
Oh snap how did that get in there? Nice catch, fixed.